### PR TITLE
Math added without casting

### PR DESCRIPTION
**Pull request by JordanCheney**

### CODE REVIEW

File: likely.cpp [hunk 1]
```diff
@@ -32,11 +32,8 @@
 #include <llvm/Transforms/Vectorize.h>
 #include <lua.hpp>
 #include <stdlib.h>
-#include <atomic>
```
Comment (jklontz) [inline likely.cpp:35]: are these not needed or something?

Comment (JordanCheney) [inline likely.cpp:35]: I believe that lua makes them obsolete, I am compiling without them.

File: likely.cpp [hunk 2]
```diff
@@ -451,6 +448,35 @@ struct MatrixBuilder
     Value *multiply(Value *i, Value *j) const { return likely_is_floating(h) ? b->CreateFMul(i, j, n) : b->CreateMul(i, j, n); }
     Value *divide(Value *i, Value *j) const { return likely_is_floating(h) ? b->CreateFDiv(i, j, n) : (likely_is_signed(h) ? b->CreateSDiv(i,j, n) : b->CreateUDiv(i, j, n)); }
 
+    Value *toFP(Value *i) const {
```
Comment (jklontz) [inline likely.cpp:452]: I'm confident there is a more elegant way to do this, but it's also bed time. Will give it more thought later :)

Comment (JordanCheney) [inline likely.cpp:452]: Sounds good see you on Monday!

File: likely.cpp (hunk 1 above)
Comment (jklontz) [inline likely.cpp:35]: :+1:

**Comment by jklontz** [comment]
This pull request seems to be missing additions to `standard.likely` for these math functions, right?

**Comment by JordanCheney** [comment]
Yep sorry, I originally updated likely_standard_library.cpp instead of standard.likely not realizing that standard.likely was the source of the cpp file. I have corrected this and will push likely.standard in the morning.

**Comment by JordanCheney** [comment]
Moved to new pull request

**Pull request closed without merging**